File: rust-sass-libsass-tests/ecosystem/smokes/perl/smoke.pl
use strict;
use warnings;
use CSS::Sass;

my $sass = CSS::Sass->new;
die "version mismatch" unless CSS::Sass::libsass_version() eq "3.6.6";
my $css = $sass->compile('$color: red; .foo { color: $color; }');
die "output mismatch: [$css]" unless $css eq ".foo {\n  color: red; }\n";
print "perl SMOKE-OK\n";

File: rust-sass-libsass-tests/ecosystem/smokes/node/smoke.js
"use strict";
const sass = require("/tmp/node-sass/lib");
const lines = sass.info.split("\n");
if (!lines[1].includes("3.6.6")) {
  throw new Error("libsass version: " + lines[1]);
}
const out = sass.renderSync({ data: "$color: red; .foo { color: $color; }" }).css.toString();
if (out !== ".foo {\n  color: red; }\n") {
  throw new Error("got: " + JSON.stringify(out));
}
console.log("node SMOKE-OK");
